.env.production

```diff
@@ -0,0 +1,9 @@
+# The port for the development server
+PORT=8080
+
+VUE_APP_WIKIBASE_API_URL='https://www.wikidata.org/w/api.php'
+VUE_APP_QUERY_SERVICE_EMBED_URL='https://query.wikidata.org/embed.html#'
+
+#Saucelabs access data - needed if testing from local-cli
+SAUCE_USERNAME='<username>'
```

Comment:
I think we can drop these `SAUCE_` keys completely from this file. We're not gaining anything from them being here with non-data.